src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/DiagramViewModel.kt

```diff
@@ -5,6 +5,7 @@ import com.structurizr.view.View
 data class DiagramViewModel(
     val key: String,
     val name: String,
+    val title: String?,
```

Comment:
So just to clarify on this one I made this decision consciously because there are already default generated names in "name" that is build automatically. Title for me was a way to override this so I've put on list of diagrams:
```
+(it.title ?: it.name)
```
So whenever title hasn't be added to view, I'm taking default name (which is `element.name + " - " + viewType` i.e. https://avisi-cloud.github.io/structurizr-site-generatr/main/internet-banking-system/container/ - here it's `Internet Banking System - Containers`, which I think is fine as a default - much better that `Untitled Diagram` given that title isn't required in DSL.  

Let me know if you want to still change that anyway.